### ISSUE TITLE
Handle logical replication sessions

### ISSUE THREAD

**Issue by frost242** [OPMDG/check_pgactivity#67]
With logical replication introduced in PostgreSQL 10, there will be active sessions running for each active publication on the sending server, as showed in pg_stat_activity : 
```
-[ RECORD 2 ]----+--------------------------------------------------
datid            | 17359
datname          | src
pid              | 13786
usesysid         | 10
usename          | thomas
application_name | local
client_addr      | ::1
client_hostname  | 
client_port      | 53628
backend_start    | 2017-01-27 18:08:19.93667+01
xact_start       | 
query_start      | 2017-01-27 18:08:19.946993+01
state_change     | 2017-01-27 18:08:19.947523+01
wait_event_type  | Client
wait_event       | WalSenderWaitForWAL
state            | active
backend_xid      | 
backend_xmin     | 
query            | walsender
```
We will have to handle this sessions correctly in backends and backends_status.

Also, we may monitor subscriptions on the receiving side with the pg_stat_subscription view. Some informations available here : https://www.postgresql.org/docs/devel/static/logical-replication-monitoring.html

**Comment by frost242** [comment]
Such sessions are not accounted as we now filter out per backend type (column was added later in pg_stat_activity).

**Comment by frost242** [comment]
Fixed in commit f4a2045
